include/z64actor.h

```diff
@@ -512,6 +512,46 @@ typedef enum {
     /* 0xFF */ NAVI_ENEMY_NONE = 0xFF
 } NaviEnemy;
 
+// A set of 4 triangles which appear around an actor when the player Z-Targets it
+typedef struct {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ f32 radius; // distance towards the center of the locked on
+    /* 0x10 */ Color_RGB8 color;
+} LockOnTriangleSet; // size = 0x14
```

Comment:
Think in general we can change "triangle set" to just "reticle" everywhere. Its a more concise word to describe what the group of triangles represents

Reply:
(also that comment on radius is an incomplete sentence. center of the locked on actor, probably?)

Reply:
I agree reticle is better. Something to change in MM later